Sources/App/configure.swift

```diff
@@ -7,13 +7,13 @@ private let jiraProjects = [
     "APPTS" : 16875, // Booking/Appointments
     "AV"    : 16942, // Core Experience / Avalon
     "CE"    : 16937, // Consultation Experience
-    // TODO: [CNSMR-2402] Re-enable CM board once they've migrated all their tickets from private instance to main one
+    // TODO: [IOSP-101/IOSP-145] Re-enable CM board once they've migrated all their tickets from private instance to main one
     // "CM"    : 16920, // Condition Management
```

Comment:
Since CM no longer exists, can we remove it?

Reply:
It was merged into Monitor which is already there. So it can be deleted

Reply:
@sdaniel55 thx. Updated the whitelist to:
 - remove commented-out old CM board
 - update some squad names in comments next to each board
 - add commented-out `TEN` board too (and created associated ticket) which I noticed was missing.

Still approved after those changes?